os/include/tinyara/mm/mm.h

```diff
@@ -661,7 +673,7 @@ int mm_size2ndx(size_t size);
 
 #ifdef CONFIG_DEBUG_MM_HEAPINFO
 /* Functions contained in kmm_mallinfo.c . Used to display memory allocation details */
-void heapinfo_parse(FAR struct mm_heap_s *heap, int mode, pid_t pid);
+void heapinfo_parse_heap(FAR struct mm_heap_s *heap, int mode, pid_t pid);
```

Comment:
why the naming has been changed?

Reply:
parse is a verb, so I explicitly make target for parsing.